src/items.rs

```diff
@@ -531,8 +529,7 @@ impl<'a> FmtVisitor<'a> {
 
                     result.push('(');
 
-                    let indent = self.block_indent + vis.len() + field.node.name.to_string().len() +
-                                 1; // Open paren
+                    let indent = self.block_indent + field.node.name.to_string().len() + "(".len();
```

Comment:
rustfmt didn't like the statement with or without line break, so I got a bit creative. It yields the same code under `-O`.